Automattic/wp-calypso
# [Coming Soon] Adjust privacy picker to contain Coming Soon option

#### Changes proposed in this Pull Request

Related to Coming Soon mode. More context available here: paObgF-MY-p2

This PR adds a Coming Soon option to privacy picker in site settings (behind a feature flag so should be good to merge). 

*Before:*

<img width="751" alt="72261218-b1e45b00-3614-11ea-9b0d-1b0ee252d852" src="https://user-images.githubusercontent.com/205419/72341143-d0f0f480-36c9-11ea-9079-9f1294ff9f07.png">

*After:*

<img width="752" alt="Zrzut ekranu 2020-01-14 o 12 25 30" src="https://user-images.githubusercontent.com/205419/72340908-57590680-36c9-11ea-89fc-2e30808fe06d.png">

**Notes:**

* This should be rebased once https://github.com/Automattic/wp-calypso/pull/38806 is merged

#### Testing instructions

* Apply D37570-code & sandbox the WPCOM REST API
* Try selecting different options in the picker and confirm they're properly stored AND reflected in the form after saving (Aside of "Coming Soon" - it's not going to work reliably just yet. That's okay for the purposes of this PR though since we're behind a feature flag).
* Repeat for Atomic site.
* Confirm you see the same form. Saving different options won't be reliable until related backend works lands - that's okay for the purposes of this diff though.
* Repeat for non-atomic Jetpack site.
* Confirm you can see the "before" privacy picker on calypso.live to confirm the feature flag is working properly.
